.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: mock-server CI
+
+on:
+  push:
+    branches: [ master, main ]
+    tags: '*'
+  pull_request:
+    branches: [ master, main ]
+
+jobs:
+  qa:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: yarn install --frozen-lockfile
+    - run: yarn lint
+    - run: yarn coverage
+    - name: Coveralls
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        path-to-lcov: './coverage/lcov.info'
+        base-path: './'
+
+  npm-publish:
+    name: npm publish
+    needs: qa
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Login to npm
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+      - name: Publish to npm
+        run: npm publish
+
+  docker-build:
+    name: Build docker
+    needs: qa
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main' }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Prepare
+        id: prep
+        run: |
+          DOCKER_IMAGE=@staticdeploy/mock-server
```

Comment:
It should be better to use github registry to upload this image

Reply:
```suggestion
          DOCKER_IMAGE=staticdeploy/mock-server
```

Reply:
```suggestion
          DOCKER_IMAGE=staticdeploy/mock-server
```